src/test/java/tempest/ui/pages/HomePageTest.java

```diff
@@ -51,4 +51,11 @@ public void chartViewButton_Sessions() {
 
         assertEquals(PageNames.CHART_VIEW, manager.getCurrentCard());
     }
+
+    @Test
+    public void manageGoalsButton(){
```

Comment:
You might already be working on this, but similar to some of the other buttons if there aren't any modules we want this button to be greyed out.

**However**, it would be sensible to wait for Ismail's branch #83 to be accepted as he's needed to change some of the methods you'd need to use _(& change)_ to implement greying out of the sessions button.

If you've already done this then maybe just copy Ismail's changes or just merge later on